notebooks/retrieve_mbox.ipynb

```diff
@@ -0,0 +1,165 @@
+{
```

Comment:
<p>@cdolfi can you create a Pipfile in your repository with all the required dependencies for this notebook and the parsing_mbox notebook?</p><p>Here's how you can use Pipenv within Jupyterhub <a href="https://docs.google.com/document/d/1BUEzAeymOr1NyWQT4_vY22dFlMinjcbeV6iFBZhBTYY/edit" target="_blank">https://docs.google.com/document/d/1BUEzAeymOr1NyWQT4_vY22dFlMinjcbeV6iFBZhBTYY/edit</a></p><p>let me know if you have any questions :)</p><p> </p><p>Also can you remove the import statements which are not being used in the notebook eg requests? maybe @sesheta bot can help with notebook linting</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/fedora-mailing-list-analysis/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>